phpcs/src/Standards/Squiz/Sniffs/PHP/DiscouragedFunctionsSniff.php

```diff
@@ -23,7 +23,7 @@ class DiscouragedFunctionsSniff extends GenericForbiddenFunctionsSniff
      * @var array<string, string|null>
      */
     public $forbiddenFunctions = [
-        'error_log' => null,
+        'error_log' => 'debugging',
```

Comment:
Uhm... and this change to the Squiz standard?

Reply:
Ups, I blindly replaced all occurences. Will revert this one.

Reply:
pong?

Reply:
I removed squiz standard change.